firefox-ios/Client/Frontend/Home/Homepage Rebuild/Merino/StoryCell.swift

```diff
@@ -68,15 +68,25 @@ class StoryCell: UICollectionViewCell, ReusableCell, ThemeApplicable, Blurrable,
         super.prepareForReuse()
         titleLabel.text = nil
         accessibilityLabel = nil
+        accessibilityHint = nil
     }
 
     // MARK: - Helpers
 
-    func configure(story: MerinoStoryConfiguration, theme: Theme) {
+    func configure(story: MerinoStoryConfiguration, theme: Theme, position: Int? = nil, totalCount: Int? = nil) {
         self.story = story
         titleLabel.text = story.title
         titleLabel.numberOfLines = getNumberOfLinesForTitle(isSponsoredStory: !story.shouldHideSponsor)
         accessibilityLabel = story.accessibilityLabel
+        if let position = position, let totalCount = totalCount {
+            accessibilityHint = String(
+                format: .FirefoxHomepage.Pocket.PositionAccessibilityHint,
+                String(position),
+                String(totalCount)
+            )
+        } else {
+            accessibilityHint = nil
```

Comment:
Do we need to set it to `nil` in an else block, if it's already set to nil from the start and in `prepareForReuse`?